packages/client/src/views/DataProvider/birth/queries.ts

```diff
@@ -145,6 +145,14 @@ export const GET_BIRTH_REGISTRATION_FOR_REVIEW = gql`
           }
           type
           timestamp
+          office {
+            name
+            alias
+            address {
+              district
+              state
+            }
+          }
```

Comment:
@tahmidrahman-dsi are these changes meant to be in this PR?

Reply:
While I was trying to fix the certificate issue, I noticed the registration location name is empty in the certificate (the `registrationLocation` handlebar is not working as it should) and I figured that recently the certificate query has been changed to `fetchBirthRegistration` from `fetchBirthRegistrationForCertificate` which had office fields. So I added these fields and now the `registrationLocation` handlebar is working.